docs/CODE_GUIDELINES.md

```diff
@@ -384,9 +384,9 @@ All component props must be defined with appropriate [TypeScript type](https://w
 
 ### Component comments
 
-All components should be documented with a comment directly before the component declaration. (It's important that there are no spaces between the comment and the component so that the comment will appear in storybook.)
```

Comment:
This doesn't seem relevant anymore because I did see a component docstring that was separated from its component by an empty space and it was fine.